src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,270 @@
+
+    ```console
+    $ xcodebuild -downloadPlatform iOS
+    ```
+
+1. Start the Simulator with the following command:
+
+    ```console
+    $ open -a Simulator
+    ```
+
+1. Set your Simulator to use a 64-bit device.
+
+   * From **Xcode**, choose a simulator device type.
+
+     1. Go to **Window** <span aria-label="and then">></span>
+        **Devices and Simulators** or press
+        <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>2</kbd>.
+
+     2. Once the **Devices and Simulators** dialog opens,
+        click **Simulators**.
+
+     3. Choose a **Simulator** from the left-hand list or
+        press **+** to create a new simulator.
+
+   * From the **Simulator** app, go to
+     **File** <span aria-label="and then">></span>
+     **Open Simulator** <span aria-label="and then">></span>
+     Choose your target iOS device.
+
+   * To check the device version in the Simulator,
+     open the **Settings** app <span aria-label="and then">></span>
+     **General** <span aria-label="and then">></span>
+     **About**.
+
+1. (_Only if needed_) If you selected a simulated
+   high-screen density iOS device,
+   it might overflow your screen. If that occurs,
+   change the presented size in the **Simulator** app.
+
+    | **Display Size**  |                          **Menu command**                          |     **Keyboard shortcut**     |
+    |:-----------------:|:------------------------------------------------------------------:|:-----------------------------:|
+    | Small             | **Window** <span aria-label="and then">></span> **Physical Size**  | <kbd>Cmd</kbd> + <kbd>1</kbd> |
+    | Moderate          | **Window** <span aria-label="and then">></span> **Point Accurate** | <kbd>Cmd</kbd> + <kbd>2</kbd> |
+    | HD accurate       | **Window** <span aria-label="and then">></span> **Pixel Accurate** | <kbd>Cmd</kbd> + <kbd>3</kbd> |
+    | Fit to screen     | **Window** <span aria-label="and then">></span> **Fit Screen**     | <kbd>Cmd</kbd> + <kbd>4</kbd> |
+    
+    {:.table .table-striped}
+
+{% endtab %}
+{% tab "Physical device" %}
+
+Set up each iOS device on which you want to test.
+
+### Configure your physical iOS device {:#attach}
+
+1. Attach your iOS device to the USB port on your Mac.
+
+1. On first connecting an iOS device to your Mac,
+   your device will display the **Trust this computer?** dialog.
+
+1. Click **Trust**.
+
+   ![Trust Mac][]
+
+### Enable Developer Mode
+
+Apple requires enabling **[Developer Mode][]**
+on the device to protect against malicious software.
+
+1. Tap on **Settings** <span aria-label="and then">></span>
+   **Privacy & Security** <span aria-label="and then">></span>
+   **Developer Mode**.
+
+1. Tap to toggle **Developer Mode** to **On**.
+
+1. Restart the device.
+
+1. When the **Turn on Developer Mode?** dialog appears,
+   tap **Turn On**.
+
+   The dialog explains that Developer Mode requires reducing
+   the security of the iOS device.
```

Comment:
Should we remove this? It feels a bit self-explanatory. It's also quite similar to this section's intro on line 135.